### PR TITLE
build(aio): do not render "constructor()" heading in API docs

### LINKED ISSUE #22363: aio + docs/api: constructor heading should match Properties and Methods headings
See @jbogarthyde's comment here https://github.com/angular/angular/pull/21874#issuecomment-363861807

### PR DESCRIPTION
**Pull request by petebacondarwin**
Closes #22363

### CODE REVIEW

**Comment by petebacondarwin** [comment]
Before:

screen shot 2018-02-22 at 21 07 50


After:

screen shot 2018-02-22 at 21 07 20

File: aio/tools/transforms/templates/api/lib/memberHelpers.html [hunk 1]
```diff
@@ -93,7 +93,7 @@ <h4 class="no-anchor">Throws</h4>
 {%- macro renderMethodDetails(methods, containerClass, itemClass, headingText) -%}
 {% if methods.length %}
 <section class="{$ containerClass $}">
-  <h2>{$ headingText $}</h2>
+  {% if headingText %}<h2>{$ headingText $}</h2>{% endif %}
```
Comment (gkalpak) [inline aio/tools/transforms/templates/api/lib/memberHelpers.html]: Are we using this macro without `headingText`? This change doesn't seem related to this PR.

**Review by gkalpak — APPROVED** [review]

File: aio/tools/transforms/templates/api/lib/memberHelpers.html (hunk 1 above)
Comment (petebacondarwin) [reply aio/tools/transforms/templates/api/lib/memberHelpers.html]: Right, sorry that slipped through from my playing around.

File: aio/tools/transforms/templates/api/class.template.html [hunk 2]
```diff
@@ -15,7 +15,9 @@
     {% include "includes/description.html" %}
     {$ memberHelpers.renderProperties(doc.staticProperties, 'static-properties', 'static-property', 'Static Properties') $}
     {$ memberHelpers.renderMethodDetails(doc.staticMethods, 'static-methods', 'static-method', 'Static Methods') $}
-    {% if doc.constructorDoc %}{$ memberHelpers.renderMethodDetail(doc.constructorDoc, 'constructor') $}{% endif %}
+    {% if doc.constructorDoc %}
+    <h2>Constructor</h2>
+    {$ memberHelpers.renderMethodDetail(doc.constructorDoc) $}{% endif %}
```
Comment (petebacondarwin) [inline aio/tools/transforms/templates/api/class.template.html]: We should still pass the cssClass as the second parameter too.
